uint64 timestamp                  # time since system start(microseconds)
uint64 timestamp_sample           # time since system start(microseconds)

float32 heading                   # Heading angle of XYZ body frame rel to NED.(radians)
float32 heading_accuracy          # Accuracy of heading of the relative position vector(radians)
